src/display/api.js

```diff
@@ -69,6 +69,11 @@ const DefaultCMapReaderFactory =
     ? NodeCMapReaderFactory
     : DOMCMapReaderFactory;
 
+/**
+ * TODO https://github.com/mozilla/pdf.js/pull/10575
```

Comment:
*Here and elsewhere:* Please ensure that you remove any TODO-comments from the final PR.

Reply:
Looking back at the original PR, this was about moving those parts to another place:

https://github.com/mozilla/pdf.js/pull/10575#discussion_r260971909

Is this still a good idea?

Reply:
No reply - removed the TODOs.